### ISSUE TITLE
Custom Softaculous UI

### ISSUE THREAD

**Issue by TristianK3604** [cdgco/VestaWebInterface#35]
I think there should be a custom UI for Softaculous and app installations too, so you don't have to link directly to the panel and Softaculous. You could use the Softaculous API for this and possibly let users add custom applications with certain parameters. Again, I could also probably help with this :smile:

**Comment by cdgco** [comment]
I am unfamiliar with the Softaculous API but will look into it. I am pretty dissatisfied with the softaculous integration with VestaCP as the process of integrating it with VWI is honestly kind of sketchy (it does require more than just linking to the softaculous page) and there are very few apps that you can install without facing a payment wall. I am unsure with the ability to add custom applications and again, this project does seem very low on the list of priorities, but I will look into some possible solutions.

**Comment by TristianK3604** [comment]
I agree. The Softaculous API is pretty frustrating, and integration too,
but with enough work and research, it can be done.
